services/ui-src/src/components/App/App.scss

```diff
@@ -1,4 +1,4 @@
-@import "~uswds/dist/css/uswds";
+@import "../../../node_modules/uswds/dist/css/uswds";
```

Comment:
changed this after the react-scripts loader had a problem loading the css file

Reply:
Requiring a package by explicitly giving the node_modules path smells really wrong, but I guess I don't see a reason why it wouldn't work... Plus I don't know the first thing about troubleshooting scss, or react-scripts 😅

Reply:
yeah I'd much prefer a real fix/answer but I believe it was just an issue with loading `css`. the next line in this file loads a `js` file using the `~` and works fine.

If anyone has ideas I'll try them!